### PR TITLE
docs: tippse from live migration meeting

### PR DESCRIPTION
**Pull request by vobu**

### CODE REVIEW

File: postgres/README.md [hunk 1]
```diff
@@ -4,6 +4,93 @@
...
+`$> cds deploy --model-only`  
+&rarr; this will create the table `cds_model` laying the foundation for the schema migration
+
+Subsequent deployments can then re-use the standard deploy mechanism via `$> cds deploy`
+
+#### On BTP, Cloud Foundry environment
+
+The above "csn-snapshots" can be implemented via the `mtar`-based approach. At the same time, the same `mtar` can be used for subsequent PostgreSQL deployments (with schema evolution).
+
+Two major steps in addition to enabling the schema evoluation are included in this `mtar`.
```
Comment (patricebender) [inline postgres/README.md:34]:
```suggestion
Two major steps in addition to enabling the schema evolution are included in this `mtar`.
```

**Review by patricebender — APPROVED** [review]
lgtm

**Comment by patricebender** [comment]
Is there something which blocks us from merging this?

**Comment by vobu** [comment]
my impression was that this served as the base content for the capire docs → IMO, could also be closed/not merged
ping @danjoa

**Comment by vobu** [comment]
let's get this bad boy in then 💨

**Pull request merged**